pkg/state/create.go

```diff
@@ -249,7 +249,7 @@ func (c *StateCreator) loadEnvValues(st *HelmState, name string, failOnMissingEn
 			}
 		}
 	} else if ctxEnv == nil && name != DefaultEnv && failOnMissingEnv {
```

Comment:
duplicate logic for env verify?

Reply:
Yeah, it's intentionally duplicated! The rationale is that changing the existing code in loadEnvValues is more likely to break helmfile v0 behavior.
In due course, we may want to fix/improve the behavior only for v1. Fixing v1 only is easier when we have two separate code for v0 and v1!

Reply:
Note that this logic in loadEnvValues is now used only in double-rendering(v0 only feature).

Reply:
ok. got it. and what should we do next for v1? @mumoshu